<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterProperties</class>
 <widget class="QDialog" name="ParameterProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parameter Properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="_typeLit">
     <property name="text">
      <string>Type:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>_type</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="_type">
     <item>
      <property name="text">
       <string>String</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bool</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Int</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Double</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>List</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="3" rowspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="_nameLit">
     <property name="text">
      <string>Name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>_name</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="_name"/>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QStackedWidget" name="_stack">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="_stringPage">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="_stringLit">
         <property name="text">
          <string>Value:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>_string</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="_string"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="_booleanPage">
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="_boolLit">
         <property name="text">
          <string>Value:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>_bool</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="_bool">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>False</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>True</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="_integerPage">
      <layout class="QFormLayout" name="formLayout_3">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::FieldsStayAtSizeHint</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="_intLit">
         <property name="text">
          <string>Value:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>_int</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="_int"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="_doublePage">
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="_doubleLit">
         <property name="text">
          <string>Value:</string>
         </property>
         <property name="buddy">
          <cstring>_double</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="_double"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="_listPage">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="_listLit">
         <property name="text">
          <string>Values:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="buddy">
          <cstring>_list</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QListWidget" name="_list"/>
       </item>
       <item row="1" column="1">
        <layout class="QVBoxLayout" name="_listButtonLyt">
         <item>
          <widget class="QPushButton" name="_listNew">
           <property name="text">
            <string>&amp;New</string>
           </property>
           <property name="shortcut">
            <string>Alt+N</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line1_3">
           <property name="frameShape">
            <enum>QFrame::HLine</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="_listUp">
           <property name="text">
            <string>Move &amp;Up</string>
           </property>
           <property name="shortcut">
            <string>Alt+U</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="_listEdit">
           <property name="text">
            <string>&amp;Edit</string>
           </property>
           <property name="shortcut">
            <string>Alt+E</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="_listDown">
           <property name="text">
            <string>Move &amp;Down</string>
           </property>
           <property name="shortcut">
            <string>Alt+D</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line1_4">
           <property name="frameShape">
            <enum>QFrame::HLine</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="_listDelete">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer3_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>24</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="_active">
     <property name="text">
      <string>Active</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>_type</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>_stack</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ParameterProperties</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ParameterProperties</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
